packages/react/src/components/ListBox/ListBox.js

```diff
@@ -47,10 +47,12 @@ const ListBox = ({
   const className = cx({
     [containerClassName]: !!containerClassName,
     [`${prefix}--list-box`]: true,
+    [`${prefix}--dropdown`]: true,
```

Comment:
should we be applying these classes to all listbox components?

Reply:
@emyarod the style rules need to be applied to all listbox components, as they're all specced to have the same opening transition. we could potentially rename the class which contains these style rules (or adding an additional class to adopt the rules) if that would be preferable?

Reply:
to maintain parity with the vanilla library we may want to use one of the alternative methods you suggested instead of reusing the dropdown class for the Downshift components